packages/fx-core/src/common/featureFlags.ts

```diff
@@ -57,7 +57,11 @@ export function isMultipleParametersEnabled(): boolean {
 }
 
 export function isOfficeXMLAddinEnabled(): boolean {
-  return isFeatureFlagEnabled(FeatureFlagName.OfficeXMLAddin, false);
+  return isFeatureFlagEnabled(FeatureFlagName.OfficeXMLAddin, true);
```

Comment:
default value of feature flag should be false

Reply:
DONE